Clients/src/pages/auth/Register.vue

```diff
@@ -0,0 +1,106 @@
+<template>
+  <div class="h-[100vh] w-full flex items-center justify-center p-24 ">
+    <form 
+      @submit.prevent="handleRegister" 
+      class="w-full max-w-md mb-6   p-16 md:p-12 flex flex-col gap-6"
+    >
+      <h1 class="text-3xl font-bold text-center text-white mb-2">Create Account</h1>
+
+      <Input
+        v-model="form.name"
+        label="Full Name"
+        type="text"
+      
+        placeholder="John Doe"
+        
+        autocomplete="name"
+        inputmode="text"
+        class="w-full"
+      />
+
+      <Input
+      v-model="form.email"
+        label="Email Address"
+        type="email"
+        placeholder="example@mail.com"
+        
+        autocomplete="email"
+        inputMode="email"
+        class="w-full"
+      />
+
+      <Input
+      v-model="form.password"
+        label="Password"
+        type="password"
+        placeholder="••••••••"
```

Comment:
Great idea, but don't you think it will be confusing a bit for the user, and he'll think there is already a password entered?
Instead we can write "Enter your password" in the placeholder.